packages/react-native/React/Fabric/Mounting/ComponentViews/Modal/RCTModalHostViewComponentView.mm

```diff
@@ -106,6 +106,7 @@ @implementation RCTModalHostViewComponentView {
   BOOL _shouldAnimatePresentation;
   BOOL _shouldPresent;
   BOOL _isPresented;
+  BOOL _modalInPresentation;
```

Comment:
if the value is always YES, why do we need to store it in a state variable?
Are there ways to modify it? 
If not, I'd rather not have the state variable (so we don't have to worry about recycling and similar stuff) and always hardcode it to YES.

Reply:
@cipolleschi The value is not always YES, it's controlled by `newProps.allowSwipeDismissal` if the allowSwipeDismissal is true we set `modalInPresentation` to false. It works the same as `_shouldAnimatePresentation` which needs to be persisted